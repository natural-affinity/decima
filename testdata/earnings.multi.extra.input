decima -e 15 20 25 45
