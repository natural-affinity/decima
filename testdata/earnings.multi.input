decima 150 119.75 29.25
