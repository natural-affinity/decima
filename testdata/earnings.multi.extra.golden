You owe: $24.00

